resources/shell/bash/ghr-completion.bash

```diff
@@ -48,13 +61,27 @@ __ghr_complete() {
     COMPREPLY=($(__ghr_complete__static "${cword}" --help))
     ;;
   open)
-    COMPREPLY=($(__ghr_complete__static "${cword}" --help))
+    if [ "$COMP_CWORD" = 2 ]; then
+      COMPREPLY=($(__ghr_complete__repos "${cword}" --help))
+    elif [ "$COMP_CWORD" = 3 ]; then
+      COMPREPLY=($(compgen -c -- "${cword}"))
+    fi
     ;;
   path)
     COMPREPLY=($(__ghr_complete__repos "${cword}"))
```

Comment:
`ghr open <tab><tab>` worked, but `ghr open siketyan<tab><tab>` did not work for me. And this is caused by `__ghr_complete__repos`, it is broken :cry:

Reply:
thats wierd 🤔 gonna look in to it

Reply:
```
$ choices=(abc bca cab)
$ compgen -W "${choices[*]}" -- "a"
abc
```

unfortunately compgen doesn't support partial matches 😢